File: 3st/Hyeonseo/programmer/이어 붙인 수.py
def solution(num_list):
    answer = 0
    even = ""
    odd = ""
    for i in num_list:
        if i % 2 == 0:
            even += str(i)
        else:
            odd += str(i)
    answer = int(even) + int(odd)
    return answer

File: 5st/gyuri/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        String str1 = "";
        String str2 = "";
        
        
        for(int i = 0; i < num_list.length; i++){
            if(num_list[i] % 2 == 1){
                str1 += num_list[i];
            } else {
                str2 += num_list[i];
            }
            
        }
            int odd = str1.isEmpty() ? 0 : Integer.parseInt(str1);
            int even = str1.isEmpty() ? 0 : Integer.parseInt(str2);
        
            answer = odd + even;
        return answer;
    }
}